group_vars/orangelight/common.yml

```diff
@@ -0,0 +1,111 @@
+ol_feedback_to: "{{ vault_ol_feedback_to }}"
+ol_feedback_cc: "{{ vault_ol_feedback_cc }}"
+ol_figgy_url: "https://figgy.princeton.edu"
+ol_honeybadger_key: "{{ vault_ol_honeybadger_key }}"
+ol_rabbit_server: "amqp://{{ ol_rabbit_user }}:{{ ol_rabbit_password }}@{{ ol_rabbit_host }}:5672"
+ol_rails_solr_url: "{{ ol_solr_url }}"
+passenger_app_root: "/opt/orangelight/current/public"
+passenger_extra_config: '{{ lookup("file", "roles/orangelight/templates/nginx_extra_config")  }}'
+pg_hba_contype: "host"
+pg_hba_method: "md5"
+pg_hba_postgresql_database: "all"
+pg_hba_postgresql_user: "all"
+pg_hba_source: "{{ ansible_host }}/32"
+postgresql_is_local: false
+postgres_version: 13
+rails_app_dependencies:
+  - libpq-dev
+  - pkg-config
+  - libtool
+  - autoconf
+rails_app_directory: "orangelight"
+rails_app_name: "orangelight"
+rails_app_symlinks: []
+rails_app_vars:
+  - name: ALMA_BIB_READ_WRITE
+    value: "{{ alma_bib_read_write_key }}"
+  - name: ALMA_BIBS_READ_ONLY
+    value: "{{ alma_bibs_read_only_key }}"
+  - name: ALMA_CONFIG_READ_ONLY
+    value: "{{ alma_config_read_only_key }}"
+  - name: ALMA_REGION
+    value: "{{ vault_alma_region }}"
+  - name: ALMA_USERS_READ_ONLY
+    value: "{{ alma_users_read_only_key }}"
+  - name: APPLICATION_HOST
+    value: "{{ ol_host_name }}"
+  - name: APPLICATION_HOST_PROTOCOL
+    value: "{{ application_host_protocol }}"
+  - name: BD_AUTH_KEY
+    value: "{{ bd_auth_key }}"
+  - name: BD_FIND_BARCODE
+    value: "{{ bd_find_barcode }}"
+  - name: BIBDATA_BASE
+    value: '{{ ol_bibdata_base }}'
+  - name: CLANCY_API_KEY
+    value: '{{ ol_clancy_api_key }}'
+  - name: CLANCY_BASE_URL
+    value: '{{ ol_clancy_base_url }}'
+  - name: FIGGY_URL
+    value: "{{ ol_figgy_url }}"
+  - name: GRAPHQL_API_URL
+    value: "{{ ol_graph_api_url }}"
+  - name: HONEYBADGER_API_KEY
+    value: "{{ ol_honeybadger_key }}"
+  - name: ILLIAD_API_KEY
+    value: '{{ vault_ol_illiad_api_key }}'
+  - name: ILLIAD_API_BASE_URL
+    value: '{{ vault_ol_illiad_api_base_url }}'
+  - name: OL_DB
+    value: "{{ ol_db_name }}"
+  - name: OL_DB_HOST
+    value: "{{ ol_db_host }}"
+  - name: OL_DB_PASSWORD
+    value: "{{ ol_db_password }}"
+  - name: OL_DB_PORT
+    value: "5432"
+  - name: OL_DB_USERNAME
+    value: "{{ ol_db_user }}"
+  - name: OL_FEEDBACK_CC
+    value: "{{ ol_feedback_cc }}"
+  - name: OL_FEEDBACK_TO
+    value: "{{ ol_feedback_to }}"
+  - name: OL_RABBIT_SERVER
+    value: "{{ ol_rabbit_server }}"
+  - name: OL_READ_ONLY_MODE
+    value: '{{ol_read_only_mode}}'
+  - name: OL_SECRET_KEY_BASE
+    value: "{{ ol_secret_key }}"
+  - name: ORANGELIGHT_ADMIN_NETIDS
+    value: '"{{ ol_admin_netids | join(" ") }}"'
+  - name: SCSB_AUTH_KEY
+    value: "{{ scsb_auth_key }}"
+  - name: SMTP_HOST
+    value: "{{ ol_smtp_host | default('localhost') }}"
+  - name: SMTP_PORT
+    value: "{{ ol_smtp_port | default('1025') }}"
+  - name: SOLR_URL
+    value: "{{ ol_solr_url }}"
+  - name: voyager_ub_id
```

Comment:
@christinach can we remove `voyager_ub_id`?

Reply:
yes we can